styles.css

```diff
@@ -263,3 +265,72 @@ button:hover {
   font-style: normal;
   font-family: 'Poppins', sans-serif;
 }
+
+.all-contact {
+  display: flex;
+  flex-direction: column;
+  background-color: #6070ff;
+  border-top-left-radius: 80px;
+}
+
+.contact-background {
+  display: flex;
+  margin-top: 21px;
+  align-items: center;
+  flex-direction: column;
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-image: url("Images/contact-form-shapes-mobile.svg");
+}
+
+.margins {
+  gap: 12px;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  padding: 82px 24px 11px 24px;
+}
+
+.all-contact h2 {
+  padding: 0;
+  margin: 0;
+  color: white;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+}
+
+.all-contact p {
+  padding: 0;
+  margin: 0;
+  font-size: 20px;
+  line-height: 28px;
+  font-weight: 400;
```

Comment:
Kindly, I suggest you use a thinner weight for the font in this section to make it more similar to the original Figma design.

Reply:
Hello @mgmediaweb 👋
The problem with the weight wasn't just in the "Contact me" section, it was actually on all of them. But thanks! I changed the font-weight directly from the **@import url** itself.